plugins/plugin-bash-like/src/pty/server.ts

```diff
@@ -275,8 +275,6 @@ export const main = async (N: number, server?: Server, preexistingPort?: number)
  */
 let count = 0
 // const children = []
-let cachedSelf
-let selfHome
```

Comment:
@starpit: The use case here is commented out, so we should decide to remove them or comment them out.

Reply:
it is fine now to remove all of this

Reply:
I think we should remove all the code which is commented in a new PR to keep the focus in the linting, which is the goal of this PR.